clang/lib/Sema/SemaBounds.cpp

```diff
@@ -5291,12 +5308,15 @@ namespace {
       }
     }
 
-    // Returns true if the expression e reads memory via a pointer.
-    // IncludeAllMemberExprs is used to modify the behavior to return true
-    // if e is or contains a pointer dereference, member reference, or
+    // Returns true if the expression e reads memory via a pointer, or e is
```

Comment:
The comment indicates that `ReadsMemoryViaPointer` returns true if `e` is null, but it actually returns false.